lib/cmds/start.ts

```
@@ -123,7 +123,13 @@ function start(options: Options) {
         path.join(outputDir, binaries[IEDriver.id].executableFilename(osType)));
   }
   if (options[Opt.EDGE]) {
-    args.push('-Dwebdriver.edge.driver=' + options[Opt.EDGE].getString());
+    // validate that the file exists prior to adding it to args
+    try {
+      let edgeFile = options[Opt.EDGE].getString();
+      if (fs.statSync(edgeFile).isFile()) {
+        args.push('-Dwebdriver.edge.driver=' + options[Opt.EDGE].getString());
+      }
+    } catch (err) { }
```

Let's add a comment here explaining why the exception is getting swallowed.